src/Codegen/Builders/ObjectBuilder.hack

```diff
@@ -46,7 +46,7 @@ final class ObjectBuilder<TField as IFieldBuilder> extends OutputTypeBuilder<\Sl
     private function generateGetFieldDefinition(HackCodegenFactory $cg): CodegenMethod {
         $method = $cg->codegenMethod('getFieldDefinition')
             ->setPublic()
-            ->setReturnType('GraphQL\\IFieldDefinition<this::THackType>')
+            ->setReturnType('?GraphQL\\IResolvableFieldDefinition<this::THackType>')
```

Comment:
Introduced `IFieldDefinition` which is a non-generic interface extended by `IResolvableFieldDefinition`. I think we won't need this once introspection is live.

Reply:
We'll need a resolver-less field definition class sooner or later anyway (e.g. interface field definitions should not have resolvers) so this is probably good.